plugin/user_plugin_server.py

```
@@ -0,0 +1,559 @@
+        return JSONResponse(resp)
+
+    except HTTPException:
+        # FastAPI 会处理
+        raise
+    except Exception as e:
+        logger.exception("plugin_trigger: unexpected error")
+        raise HTTPException(status_code=500, detail=str(e))
+
+if __name__ == "__main__":
+    import uvicorn
+    uvicorn.run(app, host="0.0.0.0", port=USER_PLUGIN_SERVER_PORT)
```

Ruff (0.14.7)

Possible binding to all interfaces

(S104)